src/pages/index.js

```diff
@@ -161,14 +213,11 @@ export default () => {
                                 Every Friday at noon EST come watch our
                                 developers make the MongoDB platform come alive.
                             </DescriptiveText>
-                            <Button
-                                secondary
-                                href="https://www.twitch.tv/mongodb"
-                                target="_blank"
-                                rel="noopener noreferrer"
-                            >
-                                Watch
-                            </Button>
+                            {twitchVideo && (
+                                <Button secondary href={twitchVideo.url}>
```

Comment:
Do we want to open this in a new tab?

Reply:
Also, do you know if we want a `noreferrer noopener` on these kinds of external links?

Reply:
I think we automatically add those `rel`s when adding a `_blank` target.